user_docs/en/changes.t2t

```diff
@@ -57,6 +57,7 @@ It is also updated with UIA enabled, when typing text and braille is tethered to
 - When using say all or commands which spell text, pauses between sentences or characters no longer gradually decrease over time. (#15739, @jcsteh)
 - NVDA no longer sometimes freezes when speaking a large amount of text. (#15752, @jcsteh)
 - More objects which contain useful text are detected, and text content is displayed in braille. (#15605)
+- When reinstalling an incompatible add-on it is no longer forcefully disabled. (#15584)

```

Comment:
Do you wish to credit yourself on this topic? I am asking since you have done so on other topics.

IMO, adding the GitHub's name in the change log helps recognize one's work, but is also useful to know who has worked on each topic, e.g. for other devs/contributors.

Reply:
Thanks, that was an oversight on my part.